onnxruntime/core/graph/function_utils.cc

```diff
@@ -316,229 +266,148 @@ std::unique_ptr<ONNX_NAMESPACE::OpSchema> CreateSchema(const std::string& functi
         ONNX_NAMESPACE::ShapeInferenceOptions options{true, 1, false};
         std::unordered_map<std::string, const ONNX_NAMESPACE::FunctionProto*> map_copy(model_local_functions.begin(),
                                                                                        model_local_functions.end());
+        std::unordered_map<std::string, TensorShapeProto> empty_map;
         ONNX_NAMESPACE::shape_inference::InferShapeForFunctionNode(*onnx_func_proto, func_domain_to_version,
-                                                                   schema_registry, ctx, options, map_copy);
+                                                                   schema_registry, ctx, options, map_copy,
+                                                                   nullptr, &empty_map);
```

Comment:
When debugging with onnx-script optional input, I am hitting symbol_table being nullptr exception. Not use the root cause if here and if this Pr solves the issue. I am urging test this code with onnx-script because now, I just merged with function-experiment with master branch and I see too many breaks :(

Reply:
@liqunfu : this PR requires the update to ONNX version. I think that might be the problem for the issues?